home/utils.py

```diff
@@ -46,53 +56,30 @@
 ]
 
 
-def import_content_csv(file, splitmessages=True, newline=None, purge=True, locale="en"):
-    def get_rich_text_body(row):
-        body = []
-        row = row.splitlines()
-        for line in row:
-            if len(line) != 0:
-                body = body + [("paragraph", RichText(line))]
-        return body
+def import_content(file, filetype, purge=True, locale="en"):
+    def get_rich_text_body(text=None):
+        if text:
+            body = []
+            lines = text.splitlines()
+            for line in lines:
+                if len(line) != 0:
+                    body = body + [("paragraph", RichText(line))]
+            return body
 
-    def get_body(raw, body_field, type_of_message):
+    def get_body(messages, type_of_message):
         struct_blocks = []
-        if "image_title" in raw and raw["image_title"]:
-            im = Image.objects.get(title=raw["image_title"]).id
-            block = blocks.StructBlock(
-                [
-                    ("image", ImageChooserBlock()),
-                ]
-            )
-            block_value = block.to_python({"image": im})
-            struct_blocks.append((type_of_message, block_value))
-
-        message_body = raw[body_field]
-        if splitmessages:
-            if newline:
-                rows = message_body.split(newline)
-                next_prompts = raw.get("next_prompt", "").split(newline)
-            else:
-                rows = message_body.splitlines()
-                next_prompts = raw.get("next_prompt", "").splitlines()
-            for i, row in enumerate(rows):
-                data = {"message": row.strip()}
-                msg_blocks = [("message", blocks.TextBlock())]
-
-                if type_of_message == "Whatsapp_Message":
-                    if len(next_prompts) > i:
-                        data["next_prompt"] = next_prompts[i]
-                        msg_blocks.append(("next_prompt", blocks.TextBlock()))
-                    elif len(next_prompts) == 1:
-                        data["next_prompt"] = next_prompts[0]
-                        msg_blocks.append(("next_prompt", blocks.TextBlock()))
-
-                if row:
-                    block = blocks.StructBlock(msg_blocks)
-                    block_value = block.to_python(data)
-                    struct_blocks.append((type_of_message, block_value))
-            return struct_blocks
-        else:
+        for raw in messages:
+            if "image_title" in raw and raw["image_title"]:
```

Comment:
SXD asked if we could use image_link so I made the export work with the link instead of the title, so there wont be an image title. We will also need the document and media